### PR TITLE
[WIP][FEATURE] Adds Use-statements to already present statements

### PR DESCRIPTION
**Pull request by tomasnorre**
Resolves issue: #2336

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Could you rebase and update tests to match it?

**Comment by TomasVotruba** [comment]
Ping when tests are updated pls

**Comment by tomasnorre** [comment]
Don't know when I'll get to it, but will let you know.

File: packages/CodingStyle/src/Application/UseImportsAdder.php [hunk 1]
```diff
@@ -77,7 +77,22 @@ public function addImportsToNamespace(
...
-        $namespace->stmts = array_merge($newUses, $namespace->stmts);
+        $wasAdded = false;
+
+        foreach ($namespace->stmts as $key => $stmt) {
+            if (! $stmt instanceof Use_) {
+                continue;
+            }
+
+            // add use statements now
+            array_splice($namespace->stmts, $key + 1, 0, $newUses);
```
Comment (TomasVotruba) [inline packages/CodingStyle/src/Application/UseImportsAdder.php]: I think 2nd and 3rd argument (`$key + 1, 0`) have to be somehow modified, to make the @Route edge case work.

**Comment by TomasVotruba** [comment]
See https://github.com/rectorphp/rector/issues/2336#issuecomment-569508961

**Pull request closed without merging**